src/Model/Subsite.php

```diff
@@ -276,7 +276,7 @@ public static function getSubsiteIDForDomain($host = null, $checkPermissions = t
 
             /** @skipUpgrade */
             $domainTableName = $schema->tableName(SubsiteDomain::class);
-            if (!in_array($domainTableName, DB::table_list())) {
+            if (!in_array(strtolower($domainTableName), array_map("strtolower", DB::table_list())) ) { // case sensitive table names
```

Comment:
This should be either
`!array_key_exists(strtolower($domainTableName))` (ideally)
or
`!(array_key_exists($domainTableName)  && in_array($domainTableName))`
as per the [return value](https://github.com/silverstripe/silverstripe-framework/blob/4/src/ORM/Connect/DBSchemaManager.php#L969) of `tableName` being keyed in the format of `[strtolower($name) => $name]`